pkg/services/store/entity/sqlstash/utils.go

```diff
@@ -33,7 +32,7 @@ func getCurrentUser(ctx context.Context) (string, error) {
 		return "", fmt.Errorf("%w: %w", ErrUserNotFoundInContext, err)
 	}
 
-	return store.GetUserIDString(user), nil
+	return user.GetUID().String(), nil
```

Comment:
@kalleep is this the string we should save?

Reply:
Yes this is the correct one. I guess we need to add the uid to id tokens to. Right now we set the id version in subject